Starwars/StarWars.cs

```diff
@@ -0,0 +1,197 @@
+            rebelss.enterStation(tie, 1);
+        }
+        catch
+        {
+            Console.WriteLine("Error: You have placed a ship in an dock number that does not exist! Please fix this and rerun.");
+            QuitRoster = true;
+        }
+        //Call out the Roster 
+        if (QuitRoster != true)
+        {
+            Console.WriteLine(rebelss.Roster);
+        }
+    }
+}
+
+//Creating a public class: Person
+public class Person
+{
+    private string firstName;
+    private string lastName;
+    private string alliance;
+    //Creating a constructor for the class Person
+    public Person(string firstName, string lastName, string alliance)
+    {
+        this.firstName = firstName;
+        this.lastName = lastName;
+        this.alliance = alliance;
+    }
+
+    //Sets up the full name based on first name and last name values
+    public string FullName
+    {
+        get
+        {
+            return this.firstName + " " + this.lastName;
+        }
+
+        set
+        {
+            string[] names = value.Split(' ');
+            this.firstName = names[0];
+            this.lastName = names[1];
+        }
+    }
+}
+
+//Creating a public class: Ship
+public class Ship
+{
+    public Person[] passengers;
+    //Creating a constructor for the class Ship
+    public Ship(string alliance, string type, int size, string shipName)
+    {
+        this.Type = type;
+        this.Alliance = alliance;
+
+        this.passengers = new Person[size];
+
+        this.shipName = shipName;
+    }
+
+//Locks in the values for the string types
+    public string Type
+    {
+        get;
+        set;
+    }
+
+    public string Alliance
+    {
+        get;
+        set;
+    }
+    public string shipName
+    {
+        get;
+        set;
+    }
+
+    public void EnterShip(Person person, int seat)
+    {
+        this.passengers[seat] = person;
+    }
+
+    public void ExitShip(int seat)
+    {
+        this.passengers[seat] = null;
+    }
+    //Change of Alliance in case of capture
+    public void allianceChange(string newAlliance)
+    {
+        this.Alliance = newAlliance;
+    }
+}
+
+//Creates the class Station
+class Station
+{
+    private string stationName;
+    private string Alliance;
+    private Ship[] shipDock;
+
+    //Constructor for Station
+    public Station(string stationName, string Alliance, int stationSize)
+    {
+        this.stationName = stationName;
+        this.Alliance = Alliance;
+        this.shipDock = new Ship[stationSize];
+    }
+    //populates the lot with the ship. Method
+    public void enterStation(Ship Ship, int lot)
+    {
+        this.shipDock[lot] = Ship;
+    }
+
+
+    //empties the lot
+    public void exitStation(int lot)
+    {
+        this.shipDock[lot] = null;
+    }
+    //Change of Alliance in case of capture
+    public void allianceChange(string newAlliance)
+    {
+        this.Alliance = newAlliance;
+    }
+
+    //Method for Roster
+    public string Roster
```

Comment:
This method is both printing out to the console and returning a value. This is not the correct thing for this method to do.